usecase/helper.go

```diff
@@ -374,8 +479,16 @@ func groupByChangeDate(parameters []orcaSchema.HistoryParameter) []GroupedHistor
 	return finalArray
 }
 
+func getMgdl(value float64) float64 {
+	return math.Round(value * MmolLToMgdLConversionFactor)
+}
+
+func getMmol(value float64) float64 {
+	return math.Round(value / MmolLToMgdLConversionFactor)
+}
+
```

Comment:
tide-v2 has this implementation in order to limit the number of digits in the float part:

```go
func convertToMmoll(value float64) float64 {
	roundedValue := math.Round(value / schema.MmolLToMgdLConversionFactor * schema.MmolLToMgdLPrecisionFactor)
	floatValue := roundedValue / schema.MmolLToMgdLPrecisionFactor
	return floatValue
}
```